landing-page/content/common/statistics-format-spec.md

```diff
@@ -0,0 +1,111 @@
+---
+url: statistics-format
+toc: false
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Plain Format for Iceberg Statistics
+
+This is a specification for the Plain Format for Iceberg Statistics, a file
+format designed to store information such as statistics about data managed in an
+Iceberg table that cannot be stored directly within the Iceberg manifest. A
+statistics file contains arbitrary pieces of information (here called "blobs"),
+along with metadata necessary to interpret them.
+
+## Format specification
+
+A file conforming to the format specification should have the structure as
+described below.
+
+### File structure
+
+The file has the following structure
+
+```
+Magic Blob₁ Blob₂ ... Blobₙ Footer
+```
+
+where
+
+- `Magic` is four bytes 0x80, 0x70, 0x73, 0x83 (short for: Plain Format for
+  Iceberg Statistics),
+- `Blobᵢ` is i-th blob contained in the file, to be interpreted by application
+  according to the footer,
+- `Footer` is defined below.
+
+### Footer structure
+
+Footer has the following structure
+
+```
+FooterPayload FooterPayloadSize Reserved FileFormatVersion Magic
+```
+
+where
+
+- `FooterPayload` LZ4-compressed, UTF-8 encoded JSON payload describing the
+  blobs in the file, with the structure described below,
```

Comment:
I think that I would prefer to allow both LZ4 and uncompressed. I doubt these are going to be very big and it may be really useful to not need to extract the payload and then uncompress it (which is hard to do on the command line).

Reply:
I think extraction alone is hard to do on command line (grab the offsets, then `dd`). do you mean inspection with `less`?
I can add Flags (don't want to repurpose Reserved yet) to store is-compressed flag, if you feel strongly about this

Reply:
Yeah, I mean inspection with less or a text editor. An uncompressed JSON blob is easy to work with.